framework/src/st_things/things_stack/things_stack.c

```diff
@@ -370,12 +375,17 @@ int things_start_stack(void)
 		THINGS_LOG_E(TAG, "Failed to initialize Cloud");
 		return 0;
 	}
-
+	
```

Comment:
please remove this white line. ( line 309, line 378, line 380, line 388)

Reply:
fixed